content/blog/2022-07-25_Galaxy_Climate_Project_Introduction_Blog_2/index.md

```diff
@@ -0,0 +1,143 @@
+
+
+
+
+
+## Climatogram
+
+![Climatogram](Galaxy_Climate_Change_Blog_2_Img4.png)
+It can be inferred from the climatogram that the climate of Mumbai is moderate and humid. Heavy rainfall in the monsoon months of June to October is clearly indicated. 
+Some rainfall is also witnessed during the winter months but it is of low intensity.
+Find the analysis in a Jupyter Notebook [here](https://github.com/Quickbeasts51429/CLIMATOGRAPH/blob/main/Climography%202%20(1).ipynb).
+
+
+## Heatmap
+
+![Surface Temperature Heatmap](Galaxy_Climate_Change_Blog_2_Img5.png)
+The above heatmap represents the [ERA-5 (fifth generation ECMWF reanalysis for the global climate and weather for the past 4 to 7 decades) Monthly mean](https://cds.climate.copernicus.eu/cdsapp#!/dataset/reanalysis-era5-single-levels-monthly-means?tab=overview) surface temperature over Mumbai for the last 65 years. The visualisation shows that the temperatures are highest near May and June. Somewhat a little high temperatures are also witnessed during October.Find the analysis in a Jupyter Notebook [here](https://github.com/NordicESMhub/galaxy-climate/blob/main/climate-heatmap/heatmap.ipynb)
+
+
+
+
+# What tool is being developed?
+
+Here comes the most awaited part, “[THE TIME SERIES EXTRACTION AND VISUALISATION TOOL](https://github.com/galaxyecology/tools-ecology/pull/58)”. So basically, I was working on the creation of a tool that extracts time series data containing a .csv file from a NetCDF file. Added to the extraction, the tool also plots a well-defined time series over a selected area and time. The tool is built on Python and wrapped in an XML wrapper class. The most intriguing and troublesome part for me was the testing. Beginning with the Windows Operating system, I faced an ocean of challenges and finally concluded that the process was not Windows’ cup of tea. I shifted to Kali Linux and did a lot of experimentation with environment creation and tool testing. Finally, my mentor Anne Fouilloux came to my rescue and the tool is now almost ready to be deployed. Huge Thanks!
+This was all for this blog, meet you in the next blog where we will be discussing the missions and organizations involved in the climate change action. You will also get to know what currently I am working on. So, don’t miss that and until then happy July!

```

Comment:
maybe happy August? :)

Reply:
Done :)

Reply:
same here, I see July still

Reply:
Please check, I guess it was some bad network connection. I've done again.